### ISSUE TITLE
[breakpoints] Standardizing

### ISSUE THREAD

**Issue by helenvholmes** [NYPL/nypl-design-system#89]
Our [current breakpoints](https://github.com/NYPL/nypl-design-system/blob/development/src/styles/00-base/base/_02-breakpoints.scss) are potentially non-standard as pointed out by Garvita in Slack [here](https://nypl.slack.com/archives/CHYEHCJTW/p1582553598002000):

```
// Avoid using in favor of atomic, content-specific, breakpoints.
// These should be used for generic code, like layouts and typography, only.
$small: 320px;
$medium: 600px;
$large: 960px;
$xl: 1280px;
// The max-width breakpoint is used when the design should be applied at whatever the
// largest breakpoint is regardless of actual pixel value. e.g. removing outer margin on body wrapper
$max-width: $xl;
```

What are industry-standard breakpoints? Can we solidify around a set that works for the entire org?

**Comment by helenvholmes** [comment]
Marking as WONTFIX and closing this out.

**Comment by helenvholmes** [comment]
Closed via #297.
